packages/public/server/src/module/Discussion/src/Controller/DiscussionController.php

```diff
@@ -105,7 +105,6 @@ public function commentAction()
             $form->setData(array_merge($this->params()->fromPost(), $data));
             if ($form->isValid()) {
                 $this->getDiscussionManager()->commentDiscussion($form);
-                $this->getDiscussionManager()->flush();
```

Comment:
Why are those removed? The flushes are usually necessary (especially for the API stuff) since they make sure that the corresponding ids exists for upcoming events.

Reply:
I moved it into the `commentDiscussion()` function at https://github.com/serlo/serlo.org/blob/a64a5629c4a47867cbfd81bd8fe6a9492323f407/packages/public/server/src/module/Discussion/src/DiscussionManager.php#L85-L95